### ISSUE TITLE
Native select examples are missing programatic labels

### ISSUE THREAD

**Issue by mfairchild365** [angular/material2#15022]
#### What is the expected behavior?
The purpose of the native select elements in the select example must be conveyed to users of screen reader software.

#### What is the current behavior?
The purpose of the native select elements in the select example is not conveyed to users of screen reader software.

#### What are the steps to reproduce?
1. Using either of the following screen readers
   - NVDA 2018.4.1 + Firefox 64
   - VoiceOver for IOS (12.1.3) + Safari
2. Open the [select example](https://material.angular.io/components/select/examples)
3. Navigate to one of the native select elements in the examples.
   - In NVDA you can press the tab key until you reach a select element
   - In VoiceOver for iOS you can swipe right until you reach a select element
4. Observe that the purpose of the select element is not conveyed by the screen reader.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
- Angular Material Example 7.2.2 located at https://material.angular.io/components/
- NVDA 2018.4.1 + FireFox 64
- VoiceOver for IOS (12.1.3) + Safari (12.1.3)

#### Is there anything else we should know?
Select elements must have a meaningful programmatic label. This does not appear to be an issue with the component itself, but rather with the way that the examples are implemented. The examples should be updated to demonstrate accessible implementations.
